src/app/features/planner/dialog-schedule-task/dialog-schedule-task.component.ts

```diff
@@ -78,6 +79,7 @@ export class DialogScheduleTaskComponent implements AfterViewInit {
     private _taskService: TaskService,
     private _reminderService: ReminderService,
     private _plannerService: PlannerService,
+    private readonly _dateAdapter: DateAdapter<unknown>,
```

Comment:
I copied the type parameterisation from other usages I saw of `DateAdapter` being injected as a dependency.